core/domain/topic_domain.py

```diff
@@ -555,13 +555,11 @@ def create_default_topic(cls, topic_id, name):
             constants.DEFAULT_LANGUAGE_CODE, 0)
 
     @classmethod
-    def update_subtopics_from_model(
-            cls, versioned_subtopics, current_version):
+    def update_subtopics_from_model(cls, versioned_subtopics, current_version):
         """Converts the subtopics blob contained in the given
         versioned_subtopics dict from current_version to
         current_version + 1. Note that the versioned_subtopics being
         passed in is modified in-place.
-
```

Comment:
@lilithxxx you should keep this blank line above "Args". I'll merge anyway but please fix this in your next PR.

Reply:
/cc @brianrodri